aries-site/src/examples/templates/forms/SettingsExample.js

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import {
+  Box,
+  Button,
+  CheckBox,
+  Form,
+  FormField,
+  Header,
+  Heading,
+  Main,
+  MaskedInput,
+  RangeInput,
+  Text,
+  TextInput,
+} from 'grommet';
+
+import { FormContainer } from '.';
+
+export const SettingsExample = () => {
+  const [formValues, setFormValues] = React.useState({
+    doNotDisturbHours: '9:00pm - 5:00am',
+    notifications: true,
+    notificationsVolume: 0.7,
+  });
+
+  // eslint-disable-next-line no-unused-vars
+  const onSubmit = ({ value, touched }) => {
+    // Your submission logic here
+  };
+
+  return (
+    <FormContainer width="medium">
+      <Box gap="medium">
+        <Header
+          direction="column"
+          align="start"
+          gap="xxsmall"
+          pad={{ horizontal: 'xxsmall' }}
+        >
+          <Heading level={3} margin="none">
+            Settings
+          </Heading>
+          <Text>for HPE Service</Text>
+        </Header>
+        <Main
+          // Padding used to prevent focus from being cutoff
+          pad={{ horizontal: 'xxsmall' }}
+        >
+          <Form
+            value={formValues}
+            onChange={setFormValues}
+            onSubmit={({ value, touched }) => onSubmit({ value, touched })}
+          >
+            <Box margin={{ bottom: 'small' }}>
+              <Text margin={{ left: 'small' }}>Key feature</Text>
+              <FormField
+                name="notifications"
+                label="Notifications"
+                component={CheckBox}
+                reverse
+                help="Description of feature"
+                toggle
+              />
+              <FormField
+                htmlFor="notificationsVolume"
+                name="notificationsVolume"
+                label="Notifications Volume"
+              >
+                <RangeInput
```

Comment:
When I hover over rangeinput knob, it creates a white border around the knob that's subtle. Is this the hover behavior we want? I'm fine with it but just couldn't tell from designs.
<img width="78" alt="Screen Shot 2020-03-05 at 9 40 52 AM" src="https://user-images.githubusercontent.com/12522275/76009114-8750a900-5ec5-11ea-83cf-b62b1ad19297.png">

Reply:
Looking at this further, we can apply hover styling in the theme, but it gets [overwritten in Grommet's implementation](https://github.com/grommet/grommet/blob/3fe8dde589f8543a9d00cf4f9951b252f8d78398/src/js/components/RangeInput/StyledRangeInput.js#L78-L91).  The style from our theme is being applied, but then gets further CSS box-shadow applied, which trumps any previous box-shadow styling.

Possible solutions:
1. Add `!important` -- which I don't recommend
2. Modify Grommet
3. Leave as is and treat as discussion point

Thoughts?

Reply:
Probably #2 but worth #3, not #1

Reply:
I would say we should leave as is for now and add as discussion point. I think the discussion should include the consideration to modify Grommet -- it seems if the user does something to the theme that they should see the effect of that.

I agree we should avoid `!important`

Reply:
Adding to discussion points in: https://github.com/hpe-design/design-system/issues/458#issue-574790708